### PR TITLE
Chore/doc filter create methods

### PR DESCRIPTION
**Pull request by unparalleled-js**
### Issues Resolved ###

- closes #

### PR Checklist ###
Did you remember to do the below?

- [ ] Add unit tests to verify this change
- [ ] Add an entry to CHANGELOG.md describing this change
- [ ] Add docstrings for any new public parameters / methods / classes

### CODE REVIEW

File: .idea/py42.iml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
+    <orderEntry type="jdk" jdkName="Python 3.7 (py3)" jdkType="Python SDK" />
```
Comment (unparalleled-js) [inline .idea/py42.iml:7]: ?

Comment (alanag13) [inline .idea/py42.iml:7]: can you undo this change and ignore this file? there is a `.gitignore` file in the .idea folder.

File: src/py42/sdk/queries/alerts/filters/alert_filter.py [hunk 2]
```diff
@@ -5,22 +5,68 @@
...
+    """Creates a :class:`FilterGroup` for filtering results where the value with key ``term``
+    contains the given value. Useful for creating ``CONTAINS`` filters that not yet supported
+    in py42 or programmatically crafting filter groups.
+
+    Args:
+        term: (str): The term of the filter, such as ``actor``.
+        value (str): The value used to match on.
+
+    Returns:
+        :class`FilterGroup`
```
Comment (alanag13) [inline src/py42/sdk/queries/alerts/filters/alert_filter.py:17]: in order for linking to properly work, you need to spell out the full path to the class. If you start the name with `~`, only the last part of the path will be displayed, e.g. `:class:`~py42.sdk.SDKClient` will appropriately link to that class, but will only display the text `SDKClient`.

**Comment by unparalleled-js** [comment]
@alanag13 Created this draft PR to have a view at a GUI diffs, it probably is not quite near ready for feedback yet

File: src/py42/sdk/queries/alerts/filters/alert_filter.py [hunk 3]
```diff
@@ -5,22 +5,68 @@
...
     filter_list = [create_query_filter(term, u"DOES_NOT_CONTAIN", value)]
     return create_filter_group(filter_list, u"AND")
 
 
 class AlertQueryFilterStringField(QueryFilterStringField):
     @classmethod
     def contains(cls, value):
+        """Creates a :class:`FilterGroup` for filtering results where the value with key
+        ``self._term`` contains the given value. Useful for creating ``CONTAINS`` filters
+        that not yet supported in py42 or programmatically crafting filter groups.
```
Comment (alanag13) [inline src/py42/sdk/queries/alerts/filters/alert_filter.py:46]: `that  *are not yet supported`

File: src/py42/sdk/queries/alerts/filters/alert_filter.py [hunk 4]
```diff
@@ -5,22 +5,68 @@
...
+    """
+
     filter_list = [create_query_filter(term, u"CONTAINS", value)]
     return create_filter_group(filter_list, u"AND")
 
 
 def create_not_contains_filter_group(term, value):
+    """Creates a :class:`FilterGroup` for filtering results where the value with key ``term``
+    does not contain the given value. Useful for creating ``DOES_NOT_CONTAIN`` filters that
+    not yet supported in py42 or programmatically crafting filter groups.
```
Comment (alanag13) [inline src/py42/sdk/queries/alerts/filters/alert_filter.py:27]: `*are not yet`

Comment (alanag13) [reply src/py42/sdk/queries/alerts/filters/alert_filter.py:27]: this is missing in most spots

File: src/py42/sdk/queries/fileevents/file_event_query.py [hunk 5]
```diff
@@ -58,16 +58,42 @@ def __iter__(self):
 
 
 def create_exists_filter_group(term):
+    """Creates a :class`FilterGroup` to find events where filter data exists.
```
Comment (alanag13) [inline src/py42/sdk/queries/fileevents/file_event_query.py:61]: These dont mention that theyre useful for when py42 does not yet support the term like the others do

File: src/py42/sdk/queries/alerts/filters/alert_filter.py (hunk 2 above)
Comment (unparalleled-js) [reply src/py42/sdk/queries/alerts/filters/alert_filter.py:17]: These ones do because they are in a different file.
